addons/point_of_sale/models/pos_config.py

```diff
@@ -70,7 +73,11 @@ def _compute_default_customer_html(self):
         'account.journal', 'pos_config_journal_rel',
         'pos_config_id', 'journal_id', string='Available Payment Methods',
         domain="[('journal_user', '=', True ), ('type', 'in', ['bank', 'cash'])]",)
-    picking_type_id = fields.Many2one('stock.picking.type', string='Operation Type')
+    picking_type_id = fields.Many2one(
+        'stock.picking.type',
+        string='Operation Type',
+        default=_default_picking_type_id,
+        domain="[('code', '=', 'outgoing'), ('warehouse_id.company_id', '=', company_id)]")
```

Comment:
required=True

Reply:
Impossible to do since the data don't and can't provide the picking_type_id because the picking types logic (creation & assignation) is handled in the post_jnit_hook (after the data).
As the picking type of PoS Configs is already required in the view, it will stay `required=False` in the model.